src/main/java/accesingdatapostgres/api/CarApi.java

```diff
@@ -0,0 +1,47 @@
+package accesingdatapostgres.api;
+
+import accesingdatapostgres.dao.entity.CarEntity;
+import accesingdatapostgres.service.CarService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+@RestController
+@RequestMapping(path = "/cars")
+public class CarApi {
+    private final CarService carService;
+
+    @Autowired
+    public CarApi(CarService carService) {
+        this.carService = carService;
+    }
+
+    @PostMapping(path = "/add")
+    public CarEntity addCar(@RequestBody CarEntity carEntity) {
```

Comment:
endpointy powinny zwracać CarDto, CarEntity to obiekt bazodanowy, nie powinno być go w tak wysokiej warstwie

Reply:
wlasnie troche mi sie pomieszalo bo poprzednio w service bylo CarDto, a teraz ma byc w api